rules/connect-client-im/rule.md

```diff
@@ -13,18 +13,22 @@ guid: 4332723e-c780-4d0c-a759-5067404ed8fc
 
 Effective communication with your clients on Microsoft Teams builds credibility, trust and ensures they receive the deserved attention.
 
+![Figure: Keeping Teams at the centre of client communication](Teams image)
+
 <!--endintro-->
 
-# Do you know why it’s important to talk with clients on Teams? 
+# Talking to clients on Teams
```

Comment:
I feel the only h1 (#) should be the title, I think a h2 here is a good replacement

From: 

# Talking to clients on Teams

To: 

## Talking to clients on Teams